README.md

```diff
@@ -96,6 +96,35 @@ This step needs root/admin privileges.
 For Ubuntus, you need to enable the universe and multiverse repositories first:
 https://help.ubuntu.com/community/Repositories/Ubuntu#Adding_Repositories_in_Ubuntu
 
+Ubuntu 24.04
+-------------------------
+This version of Ubuntu has only libwxgtk3.2 and it requires to build wxGTK manualy
```

Comment:
Did you check how far is 3.2 from 3.0 API-wise? Would it be difficult to fix OpenASIP code base to support it? Unfortunately 20.04 nor 22.04 do not support it, so we'd need to #ifdef-support the both versions.